cpmpy/solvers/choco.py

```diff
@@ -575,6 +575,21 @@ def _get_constraint(self, cpm_expr):
                 chc_star = min(np.nanmin(table), *get_bounds(array)[0]) -1
                 chc_table = np.nan_to_num(table, nan=chc_star).astype(int).tolist()
                 return self.chc_model.table(self.solver_vars(array), chc_table, universal_value=chc_star, algo="STR2+")
+            elif cpm_expr.name == "regular":
+                from pychoco.objects.automaton.finite_automaton import FiniteAutomaton
+                array, transitions, start, accepting = cpm_expr.args
+                for i, (lb, ub) in enumerate(zip(*get_bounds(array))):
+                    if lb < 0 or ub > 65535:
```

Comment:
we should move magic numbers into constants at the top of the file, imo. perhaps out of scope or perhaps easy to quickly fix?